generators/app/README.md

```diff
@@ -0,0 +1,79 @@
+# Polymer App Toolbox - Starter Kit
+
+[![Build Status](https://travis-ci.org/PolymerElements/polymer-starter-kit.svg?branch=master)](https://travis-ci.org/PolymerElements/polymer-starter-kit)
+
+This template is a starting point for building apps using a drawer-based
+layout. The layout is provided by `app-layout` elements.
+
+This template, along with the `polymer-cli` toolchain, also demonstrates use
+of the "PRPL pattern" This pattern allows fast first delivery and interaction with
+the content at the initial route requested by the user, along with fast subsequent
+navigation by pre-caching the remaining components required by the app and
+progressively loading them on-demand as the user navigates through the app.
+
+The PRPL pattern, in a nutshell:
+
+* **Push** components required for the initial route
+* **Render** initial route ASAP
+* **Pre-cache** components for remaining routes
+* **Lazy-load** and progressively upgrade next routes on-demand
+
+### Setup
+
+##### Prerequisites
+
+First, install [Polymer CLI](https://github.com/Polymer/polymer-cli) using
+[npm](https://www.npmjs.com) (we assume you have pre-installed [node.js](https://nodejs.org)).
+
+    npm install -g polymer-cli
+
+And install the dependencies:
+
+    npm install && bower install
+
+### Start the development server
+
+This command serves the app at `http://localhost:8080` and provides basic URL
+routing for the app:
+
+    polymer serve --open
+
+### Build
+
+The included `gulpfile.js` relies on [the `polymer-build` library](https://github.com/Polymer/polymer-build),
+the same library that powers Polymer CLI. Out of the box it will clean the
+`build` directory, and provide image minification. Follow the comments in the
+`gulpfile.js` to add additional steps like JS transpilers or CSS preprocessors.
+
+Also, generates a service-worker.js file with code to pre-cache the dependencies
+based on the entrypoint and fragments specified in `polymer.json`.
+
+    npm run build
+
+### Preview the build
+
+This command serves the minified version of the app at `http://localhost:8080`:
+
+    polymer serve build/
+
+### Run tests
+
+This command will run [Web Component Tester](https://github.com/Polymer/web-component-tester)
+against the browsers currently installed on your machine:
+
+    polymer test
+
+### Adding a new build step
+
+The `gulpfile.js` already contains an example build step that demonstrates how
+to run image minification across your source files. For more examples, refer to
+the section in [the polymer-build README on extracting inline sources](https://github.com/Polymer/polymer-build#extracting-inlined-cssjs).
+
+### Adding a new view
+
+You can extend the app by adding more views that will be demand-loaded
+e.g. based on the route, or to progressively render non-critical sections of the
+application. Each new demand-loaded fragment should be added to the list of
+`fragments` in the included `polymer.json` file. This will ensure those
```

Comment:
Reword:

"This will ensure those components and their dependencies are added to the list of pre-cached components and will be included in the `bundled` build."